features/google/src/components/awakeningLocationMinistory/awakeningLocationMinistory.jsx

```diff
@@ -0,0 +1,135 @@
+import "./awakeningLocationMinistory.css";
```

Comment:
He, we had a small discussion with @stephentattum where he suggested "waking up" instead of awakening, which should be reserved only for epiphanies and other revelations about data sovereignity :rofl:

Reply:
Yeah that's true there's a quite a bit of gravity to awakening haha